### ISSUE TITLE
Support for configurable Compose file version

### ISSUE THREAD

**Issue by dalberto** [buildkite-plugins/docker-compose-buildkite-plugin#37]
I'm attempting to use this plugin to run a docker-compose pipeline. The application was configured to use docker-compose prior to the experiment with Buildkite. In order to support environment variable interpolation *with* defaults in the compose file, the application uses the 2.1 format. Unfortunately, it looks like this causes an error similar to the following:

```
ERROR: Version mismatch: file ./docker-compose.yml specifies version 2.1 but extension file ./docker-compose.buildkite-web-override.yml uses version 2.0
```

One potential solution is to support an environment variable that specifies the docker-compose version (which itself can default to 2.0 😄 ).  I imagine this would make it easier for existing applications with significant compose configurations to get onboard.

Another solution is to add an environment variable by disables the override altogether. This solution might be trickier, since (judging from `build.sh`), there would be no guarantee that the image is correctly tagged for pushing.

**Comment by lox** [comment]
Ugh, sorry about that! Very annoying that they don't support 2.0 + 2.1 given the subtle differences. 

I think it would make sense to detect the version of the file and match that in the override. We'd love a PR, otherwise I can look at this when I get a chance.

**Comment by lox** [comment]
Fixed in #40! Thanks!
